sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -343,6 +344,33 @@ protected AnalyzedRelation visitQuerySpecification(QuerySpecification node, Stat
         AnalyzedRelation relation;
         if (context.sources().size() == 1) {
             AnalyzedRelation source = Iterables.getOnlyElement(context.sources().values());
+
+            // Queries on concrete table relation must result in a QueriedTable so that the query spec will be handled
+            // on the collect phase instead of a generic filter on top of it. Also the logical planner will do a GET
```

Comment:
>  Queries on concrete table relation must result in a QueriedTable so that the query spec will be handled on the collect phase instead of a generic filter on top of it

I think with the push down rules that work on the logical operators where the `QueriedTable` is not relevant anymore this isn't true anymore?

Only the second part about the `Get` ?

Reply:
right, I was confused, will change ;)